src/components/transformation/create/legacy/LanguageSelector.tsx

```diff
@@ -22,7 +22,7 @@ function LegacyLanguageSelector() {
     const setLanguage = useTransformationCreate_setLanguage();
 
     return (
-        <LegacyStepWrapper last>
+        <LegacyStepWrapper last="true">
```

Comment:
Fixes dom related error as `last` was actually added to the dom and mui was complaining.